cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -1520,28 +1520,43 @@ private Map<ProgramRunId, RunRecordDetail> getUnfinishedRuns(Set<ProgramRunId> p
     List<List<Field<?>>> allKeys = new ArrayList<>();
     for (ProgramRunId programRunId : programRunIds) {
       allKeys.add(getProgramRunInvertedTimeKey(TYPE_RUN_RECORD_ACTIVE, programRunId,
-                                               RunIds.getTime(programRunId.getRun(), TimeUnit.SECONDS)));
+                                               RunIds.getTime(programRunId.getRun(), TimeUnit.SECONDS),
+                                               false));
```

Comment:
Why skipping version here?

Reply:
Full reasoning here: https://github.com/cdapio/cdap/pull/14605#issue-1401576442

put it short: we have runID as UUID, we don't need version and in current LogHTTPHandler there's no way to pass in the version info